python/custom/reports/per_line_mcdc_report.py

```diff
@@ -0,0 +1,8 @@
+from vector.apps.ReportBuilder.custom_report import CustomReport
```

Comment:
Remove this for now; we'll add it it back in a later PR

Reply:
Removed in b47303ebd7cd9ac952b379fb59fcb96cd0b4c57d